apps/condo/domains/analytics/access/OverviewDashboardService.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Generated by `createservice analytics.OverviewDashboardService`
+ */
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+
+const { checkUserBelongsToOrganization, checkOrganizationPermission } = require('@condo/domains/organization/utils/accessSchema')
+
+async function canOverviewDashboard ({ authentication: { item: user }, args: { data: { where: { organization } } } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin || user.isSupport) return {}
+
+    const userBelongsToOrganization = checkUserBelongsToOrganization(user.id, organization)
+    if (!userBelongsToOrganization) return false
+
+    const canManageOrganization = await checkOrganizationPermission(user.id, organization, 'canManageOrganization')
+
+    return !!canManageOrganization
```

Comment:
What about access of invited employee?

Reply:
I have no clue what did you mean. By design only organization administrator role has access to this data